src/main/java/appeng/parts/automation/PartLevelEmitter.java

```diff
@@ -370,16 +380,60 @@ public boolean isValid( final Object effectiveGrid )
 		}
 	}
 
-	@Override
-	public void postChange( final IBaseMonitor<IAEItemStack> monitor, final Iterable<IAEItemStack> change, final BaseActionSource actionSource )
-	{
-		int currentTick = MinecraftServer.getServer().getTickCounter();
-		if (currentTick - lastWorkingTick > AEConfig.instance.levelEmitterDelay)
-		{
-			this.updateReportingValue((IMEMonitor<IAEItemStack>) monitor);
-			lastWorkingTick = currentTick;
-		}
-	}
+    @Override
```

Comment:
I don't care about braces and whitespaces, but at least use the same indentation

Reply:
Ah it's editorconfig making it spaces, while the original code is tabs, I'll fix this